### PR TITLE
CSP config option

### PR DESCRIPTION
**Pull request by chereseeriepa**
Allows the option to configure the CSP.

@staltz I suppose you'd want me to remove `lodash.get`, what would you suggest?

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -19,6 +19,7 @@ const createUnboxTransform = (queryParam) => {
 
 function ServeBlobs(sbot, config) {
   const corsEnabled = get(config, 'serveBlobs.cors', false)
+  const blobCSP = get(config, 'serveBlobs.csp', 'default-src none; sandbox')
```
Comment (mixmix) [inline index.js:22]: It might be good to extract all default config into one place like `default-config.js`

then pull into that : port, cors, csp

**Review by mixmix — APPROVED** [review]
This looks clear to me. Keen to wait until we get input from @staltz 

For the moment just point repos at this branch if you need to use this

**Comment by staltz** [comment]
Good PR. I'll merge with a squash to convert the two commits (same message for both?) into one.

It would be good to get a separate PR where lodash.get is replaced by plain javascript. Here's a suggestion how to do that:

```diff
-const corsEnabled = get(config, 'serveBlobs.cors', false)
+const corsEnabled = config && config.serveBlobs && typeof config.serveBlobs.cors === 'boolean' 
+  ? config.serveBlobs.cors 
+  : false
```

Note (just for curiosity), if this repo had TypeScript or Babel, we could write this instead:

```diff
-const corsEnabled = get(config, 'serveBlobs.cors', false)
+const corsEnabled = config?.serveBlobs?.cors ?? false
```

**Comment by staltz** [comment]
Released 2.4.0

**Pull request merged**